server/auth_middleware_test.go

```diff
@@ -317,7 +317,7 @@ func TestAuthorizationMiddleware(t *testing.T) {
 				w.WriteHeader(http.StatusOK)
 			})
 
-			authHandler := testServer.Authorization(handler)
+			authHandler := testServer.Authorization(handler, nil)
```

Comment:
can you add some test case for a no auth url?

Reply:
I have it in another commit. Lemme push it here.

Reply:
updated, with the code to skip RBAC for ACM queries too